examples/plot_tabular_regression.py

```diff
@@ -0,0 +1,154 @@
+"""
+Tabular Regression with scikit-learn
+-------------------------------------
+
+This example shows how you can use skops' capabilities to serialize a model, create a model card and a Hugging Face Hub repository for a
+tabular regression task using scikit-learn. We also show how you can generate
+a model card for the model and the task at hand.
+"""
+
+# %%
+# Imports
+# =======
+# First we will import everything required for the rest of this document.
+
+from pathlib import Path
+from tempfile import mkdtemp, mkstemp
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import sklearn
+from sklearn.datasets import load_diabetes
+from sklearn.linear_model import LinearRegression
+from sklearn.metrics import mean_absolute_error, mean_squared_error, r2_score
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+
+import skops.io as sio
+from skops import card, hub_utils
+
+# %%
+# Data
+# ====
+# We will use diabetes dataset from sklearn.
+
+X, y = load_diabetes(return_X_y=True)
+X_train, X_test, y_train, y_test = train_test_split(
+    X, y, test_size=0.2, random_state=42
+)
+
+# %%
+# Train a Model
+# =============
+# To train a model, we need to convert our data first to vectors. We will use
+# StandardScalar in our pipeline. We will fit a Linear Regression model with the outputs of the scalar.
+model = Pipeline(
+    [
+        ("scaler", StandardScaler()),
+        ("linear_regression", LinearRegression()),
+    ]
+)
+
+model.fit(X_train, y_train)
+
+# %%
+# Inference
+# =========
+# Let's see if the model works.
+prediction_data = X_test[:5]
+prediction = model.predict(prediction_data)
+print(prediction)
```

Comment:
```suggestion
y_pred = model.predict(X_test[:5])
print(y_pred)
```

This is not a must, but I would prefer we use the canonical `y_pred` for the output of `model.predict` (further below, we also use `y_pred`). I know that in `plot_text_classification.py`, we also don't use `y_pred`, but I'd rather change it there as well, than keeping it here.